src/menu/actions.c

```diff
@@ -81,7 +94,16 @@ static void actions_update_direction (menu_t *menu) {
 }
 
 static void actions_update_buttons (menu_t *menu) {    
-    joypad_buttons_t pressed = joypad_get_buttons_pressed(JOYPAD_PORT_1);
+    int prev_pad = -1;
+    joypad_buttons_t pressed = {0};
+
+    JOYPAD_PORT_FOREACH (i) {
+        joypad_buttons_t pressed_temp = joypad_get_buttons_pressed(i);
+        if ((pressed_temp.raw || prev_pad == i) && pressed_temp.raw != pressed.raw) {
```

Comment:
Don't quite understand why is this check `prev_pad == i` here. `prev_pad` is always `-1` and `i` value will never be equal to `-1`.

Reply:
Same thing with the `pressed_temp.raw != pressed.raw` check. It's comparing to zero and the first check `if ((pressed_temp.raw...` does the same thing.